src/map/script.c

```diff
@@ -20991,6 +20991,9 @@ BUILDIN(instance_create)
 	const char *name;
 	int owner_id, res;
 	int type = IOT_PARTY;
+	struct map_session_data *sd = script->rid2sd(st);
```

Comment:
this change not allow use instance_create without attached player to npc.
Probably need use map->id2sd and script->rid2sd in same way as in strcharinfo?